### PR TITLE
Make see also match guidelines

### PR DESCRIPTION
**Pull request by Josh-Cena**

### CODE REVIEW

File: files/en-us/web/javascript/equality_comparisons_and_sameness/index.md [hunk 1]
```diff
@@ -224,4 +224,4 @@ console.log(f2b(nan2)); // Uint8Array(8) [1, 0, 0, 0, 0, 0, 248, 127]
 
 ## See also
 
-- [JS Comparison Table](https://dorey.github.io/JavaScript-Equality-Table/)
+- [JS Comparison Table](https://dorey.github.io/JavaScript-Equality-Table/) by dorey
```
Comment (hamishwillee) [inline files/en-us/web/javascript/equality_comparisons_and_sameness/index.md:227]: Perhaps make it clear it's a Github handle? 
```suggestion
- [JS Comparison Table](https://dorey.github.io/JavaScript-Equality-Table/) by [dorey](https://github.com/dorey)
```

File: files/en-us/web/javascript/guide/regular_expressions/groups_and_backreferences/index.md [hunk 2]
```diff
@@ -190,12 +190,8 @@ console.log(lines.join("\n"));
 
 ## See also
 
-- A polyfill of [`RegExp` named capture groups](https://github.com/zloirock/core-js#ecmascript-string-and-regexp) is available in [`core-js`](https://github.com/zloirock/core-js)
```
Comment (hamishwillee) [inline files/en-us/web/javascript/guide/regular_expressions/groups_and_backreferences/index.md:193]: We're at the point we no longer need this polyfill?

File: files/en-us/web/javascript/reference/errors/no_variable_name/index.md [hunk 3]
```diff
@@ -96,6 +96,6 @@ const arr = [1, 2, 3, 4, 5];
 
 ## See also
 
-- [Good variable names](https://wiki.c2.com/?GoodVariableNames)
+- [Lexical grammar](/en-US/docs/Web/JavaScript/Reference/Lexical_grammar)
 - {{jsxref("Statements/var", "var")}}
-- [Variable declarations in the JavaScript Guide](/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#declarations)
+- [Grammar and types § Variable declarations](/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#declarations)
```
Comment (hamishwillee) [inline files/en-us/web/javascript/reference/errors/no_variable_name/index.md:101]: I don't know this `§` character, which to me means that it is not standard enough for this use.
I tend to use this, though note it has been "corrected" by others.
```suggestion
- [Grammar and types > Declarations](/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#declarations)
```

Comment (hamishwillee) [reply files/en-us/web/javascript/reference/errors/no_variable_name/index.md:101]: I guess you could also do "Declarations (in Grammar and types)"

File: files/en-us/web/javascript/reference/errors/redeclared_parameter/index.md [hunk 4]
```diff
@@ -63,4 +63,4 @@ function g(arg) {
 - {{jsxref("Statements/let", "let")}}
 - {{jsxref("Statements/const", "const")}}
 - {{jsxref("Statements/var", "var")}}
-- [Declaring variables](/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#declarations) in the [JavaScript Guide](/en-US/docs/Web/JavaScript/Guide)
+- [Grammar and types § Declaring variables](/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#declarations)
```
Comment (hamishwillee) [inline files/en-us/web/javascript/reference/errors/redeclared_parameter/index.md:66]: Ditto re §

Comment (hamishwillee) [reply files/en-us/web/javascript/reference/errors/redeclared_parameter/index.md:66]: And everywhere it is used.

File: files/en-us/web/javascript/guide/regular_expressions/groups_and_backreferences/index.md (hunk 2 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/guide/regular_expressions/groups_and_backreferences/index.md:193]: This is a guide article. I've moved the polyfill link to its reference.

File: files/en-us/web/javascript/reference/errors/no_variable_name/index.md (hunk 3 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/errors/no_variable_name/index.md:101]: § is pretty standard, just like &, it has one clear meaning. And it's odd enough that if you don't know it you would look it up and understand its meaning 😃

`>` has the ambiguity that you don't know if this means "a doc page among a group of pages" (e.g. "Guide > Grammar and types") or "a section within one page".

Comment (hamishwillee) [reply files/en-us/web/javascript/reference/errors/no_variable_name/index.md:101]: > § is pretty standard—doesn't necessarily mean it's very popular—but just like &, it has one clear meaning. And it looks odd enough that if you don't know it you would look it up and understand its meaning 😃

I looked it up and got no hits on google on three searches - "what is  §","what is  § symbol", ""what is  § character". I still don't know. 
It's too far out of normal use to be worth using.

I'm sure we can find a better standard way to reference this - such as "Declarations (in Grammar and types)" or "Grammar and types: declaration"

Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/errors/no_variable_name/index.md:101]: This is really interesting—if you search for "§" by itself you get a lot of results. My guess is that "§" is deemed as a trash character when there are other words.

Comment (hamishwillee) [reply files/en-us/web/javascript/reference/errors/no_variable_name/index.md:101]: Yes, I think it's confusing. If you love this, then you could raise a discussion in community to agree "the right way". If that ends up being this character then I'd have to learn to love it :-)

**Comment by Josh-Cena** [comment]
@hamishwillee I've got rid of § in this PR. I realized they can be safely omitted altogether, pushing this problem to a future me to worry about.

**Comment by hamishwillee** [comment]
@Josh-Cena Curses - I was hoping you'd chase down the "standard" way to reference a section :-).

This looks good. merging.

**Review by hamishwillee — APPROVED** [review]

**Pull request merged**